### DISCUSSION TITLE
Can we change the slider button to icons?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#2065] [Q&A] [answered]
### Question

Hi All,

I am trying to create something like this image below:
![image](https://github.com/zauberzeug/nicegui/assets/65127974/a617a79b-6c42-4a28-8959-73cd3d644775)

I was trying to go through the props under QSlider but could not figure out how to change the slider button into a different shape or replace it with some icon. Is this possible?

I saw there is a prop called ```thumb-path``` for custom svg, so I tried to use my own svg logos like this
```python
ui.slider(min=1,max=10,step=1,value=2).classes('pt-4').props('vertical thumb-size=30px thumb-path="/images/test/fb.svg"')
```
But this does not work, I guess this is not the right way to use that prop?

Can you please advise on this, how to use my custom icons/shapes in the slider?

Thanks,
Anindya

**Comment by falkoschindler** [comment, marked as answer]
I think "tumb-path" expects a plain SVG path, e.g. "M5 5 h10 v10 h-10 v-10". So it doesn't look like Quasar has an API for changing the thumb image. But you can use some custom CSS as a workaround:

```py
ui.add_head_html('''
    <style>
        svg.q-slider__thumb-shape {
            background-image: url(https://cdn.iconscout.com/icon/free/png-512/free-facebook-75-189796.png);
            background-size: 100%;
        }
        svg.q-slider__thumb-shape path {
            display: none;
        }
    </style>
''')
ui.slider(min=0, max=100, step=1, value=50)
```

<img width="489" alt="Screenshot 2023-11-22 at 14 28 58" src="https://github.com/zauberzeug/nicegui/assets/5767091/34871b0c-27e5-474d-ba77-c3487263b59e">

**Reply by Anindya088** [reply]
@falkoschindler  this is so cool :) Thanks a lot, I can use this approach for my case.
